open_spiel/python/algorithms/external_sampling_mccfr_test.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for open_spiel.python.algorithms.cfr."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl.testing import absltest
+import numpy as np
+from open_spiel.python import policy
+from open_spiel.python.algorithms import exploitability
+from open_spiel.python.algorithms import external_sampling_mccfr
+import pyspiel
+
+SEED = 39823987
+
+
+class OutcomeSamplingMCCFRTest(absltest.TestCase):
```

Comment:
Rename to ExternalSamplingMCCFRTest

Reply:
Yes, good point. Forgot to change the method names!